lib/gyp/bindings.js

```diff
@@ -5,6 +5,13 @@ const fs = require('fs');
 const execSync = require('child_process').execSync;
 const mkdirpSync = require('mkdirp').sync;
 
+if (process.platform === 'win32') {
+  const getVS2017Path = require('get-vs2017-path').getVS2017Path;
+  exports.getVS2017Path = function (hostBits, target_arch) {
+    return getVS2017Path(hostBits, target_arch, exports);
```

Comment:
I name it "require less" I use your external bindings (`execSync`, `log`, and `path`)